bindings/CMakeLists.txt

```diff
@@ -1,3 +1,12 @@
 add_subdirectory(c)
+add_subdirectory(flow)
 add_subdirectory(python)
-add_subdirectory(java)
+if(BUILD_JAVA)
```

Comment:
It'd be nice to be consistent with the other bindings here. Change to `WITH_JAVA`?